Simpson_Assign1/Program.cs

```diff
@@ -16,11 +16,131 @@ static void Main(string[] args)
             Students = InitializeStudents();
             Courses = InitializeCourses();
 
+            string input = "";
             do
             {
+                Console.WriteLine("\nPlease choose from the following options:\n");
                 PrintMenu();
+                input = Console.ReadLine();
+
+                int count = 0;
+                switch (input)
+                {
+                    case ("1"):
+                        Console.WriteLine("\nStudent List <All Students>:");
+                        Console.WriteLine("------------------------------------------------------");
+                        foreach (Student eachStudent in Students)
+                        {
+                            Console.WriteLine(eachStudent.ToString());
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("2"):
+                        Console.Write("Which major list would you like printed?");
+                        string major = Console.ReadLine();
+                        Console.Write("\nStudent List <{0} Majors>", major);
+                        Console.WriteLine("\n------------------------------------------------------");
+                        List<Student> foundStudent = Students.FindAll(x => x.Major == major);
+                        foreach (Student eachStudent in foundStudent)
+                        {
+                            Console.WriteLine(eachStudent.ToString());
+                            count++;
+                        }
+                        if (count == 0)
```

Comment:
you can do:
if (foundStudent.Count == 0)

or you could also do:
if (!foundStudent.Any())

The first compares the list count to 0.
The second just says "If the list doesn't have any items"